CalendarFXGoogle/src/main/java/module-info.java

```diff
@@ -3,7 +3,7 @@
 
     requires org.kordamp.ikonli.javafx;
     requires org.kordamp.ikonli.fontawesome;
-    requires org.kordamp.iconli.core;
+    requires org.kordamp.ikonli.core;
```

Comment:
Doesn't this mean we also have to update the version of ikonli?

Reply:
Which is fine with me of course.

Reply:
To be honest, we thought that it's a typo - it prevented the code from compiling. In other places the `ikonli` phrase was being used - so we decided to use `ikonli` here as well.
Now I can see that there is an issue reported regarding this problem: https://github.com/dlsc-software-consulting-gmbh/CalendarFX/issues/122 - so probably it should fix it as well.